config/schemes/semantic_artefact_catalog.yml

```diff
@@ -63,15 +63,28 @@ fundedBy:
   helpText: "Founder of the catalog"
   example: ''
   enforcedValues: [img_src: "", url: "" ]
-  default: []
+  default: [
```

Comment:
you need to update or remove https://github.com/ontoportal-lirmm/ontologies_linked_data/blob/master/config/config.rb.sample#L96-L127

Reply:
Done